client/src/partials/templates/bhLocationSelect.tmpl.html

```diff
@@ -3,11 +3,11 @@
   a child form is added for validation purposes, and to make the HTML easier
   to read.
 -->
-<fieldset ng-disabled="$ctrl.disable" ng-form="LocationSelectForm" data-bh-location-select>
+<fieldset ng-disabled="$ctrl.disable"  ng-form="{{$ctrl.name}}" ng-model="somevaluetwo" data-bh-location-select>
 
   <!-- allows a user to select a country -->
   <div class="form-group"
-    ng-class="{ 'has-error' : $ctrl.validationTrigger && LocationSelectForm.country.$invalid }"
+    ng-class="{ 'has-error' : $ctrl.validationTrigger && this[$ctrl.name].country.$invalid }"
```

Comment:
`ng-init` is an interesting idea although I have found that it runs before Angular completes processing the `ng-form` directive. This means that whatever you assign from `this` will at the time be undefined. 

A similar principle applies for setting this alias up in the controller - the only method I'd found to make the alias work was to use a `$timeout` to ensure that compile/ linking process has taken place. 
```js
$timeout(() => vm.LocationForm = $scope[vm.name])
```

Have you made `ng-init` work in a similar case? There may be something I'm missing.